libs/core/src/lib/services/usb.serial.service.ts

```diff
@@ -8,10 +8,19 @@ export class UsbSerialService {
     protected static instance;
 
     /**
-     * Get instance
-     * @returns
+     * Get instance, get the instance of the usb serial service.
+     * @returns an instance of the usb serial service
+     * @deprecated Use the new {@link createInstance} instead.
      */
     public static getInstance() {
+        UsbSerialService.createInstance();
+    }
+
+    /**
+     * Create instance, create the instance of the usb serial service.
+     * @returns an instance of the usb serial service
+     */
+    public static createInstance() {
```

Comment:
Add a return type `UsbSerialService` otherwise it will be considered as `any`. Same for `getInstance`